src/core/thread/mesh_forwarder.cpp

```diff
@@ -1070,6 +1086,8 @@ ThreadError MeshForwarder::HandleFrameRequest(Mac::Frame &aFrame)
     if (mSendMessage == NULL)
     {
         SendEmptyFrame(aFrame);
+        aFrame.SetIsARetransmission(mSendMessageIsARetransmission);
```

Comment:
Actually had this as `false` initially but changed it to the `mSendMessageIsARetransmission`...
I agree setting it to `false` may be cleaner and more readable....